CHANGELOG.md

```diff
@@ -17,152 +19,222 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.12.0] - 2023-07-26
+
+### Added
+
+* Added option in `.to_pandas()` method to include expanding metadata columns into a dataframe as well as an option to prefix those columns. Default is to prepend ".metadata" and `expand_metadata` defaults to `False` in order to maintain backwards compatibility.
+* Updated installation process for contributing.
```

Comment:
Perhaps use `-` as all other entries do this 😅

Reply:
Again, must have been my editor. I fix.